mentoring/tests/integration/base_test.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014 Harvard
+#
+# Authors:
+#          Xavier Antoviaque <xavier@antoviaque.org>
```

Comment:
@bradenmacdonald I'm fine with moving the list of authors to a text file, it's true that this tends to get out of date at this place. Git also provides the info, but I feel it's nice to formally give credit to all the contributors of a repository.